scripts.js

```diff
@@ -115,5 +114,23 @@ const inventorFirstAndLastNames = () => {
   });
 };
 
+// Array.prototype.sort()
+// 3. Sort the inventors by birthdate, oldest to youngest
+const inventorBirthdate = () => {
```

Comment:
How could this name be more descriptive? This suggests that it returns the birth date, but that's not what it does. Perhaps `sortByBirthDates` or similar